doc/cascadia/profiles.schema.json

```diff
@@ -869,15 +930,37 @@
         "icon": { "$ref": "#/definitions/Icon" },
         "name": {
           "description": "The name that will appear in the command palette. If one isn't provided, the terminal will attempt to automatically generate a name.",
+          "properties":{
+            "key":{
+              "type":"string"
+            }
```

Comment:
Can you document that this is for localization? <https://github.com/microsoft/terminal/blob/2531454bcd46eadc14273ca85acc76231b8810cb/src/cascadia/TerminalSettingsModel/Command.cpp#L199-L203>